tests/test_electric_bus.py

```diff
@@ -17,20 +17,27 @@
 
 
 class TestMILPbus(TestCase):
-    """MILP problem test and NLP problem test. In same test to not duplicate code"""
-
     def test_voltages_and_power_network1(self):
-        """Test the voltage levels relative to each other. The absolute value
-        (230V for instance) is determined by other components than the bus"""
+        """
+        This test checks the behaviour of electricity networks with an bus asset. A bus asset is the
```

Comment:
"Test the behaviour ..." or "Check the behaviour ..."

Reply:
done